management/ia/templates_test.go

```diff
@@ -371,60 +371,145 @@ func TestListTemplate(t *testing.T) {
 	require.NoError(t, err)
 	defer deleteTemplate(t, client, name)
 
-	resp, err := client.ListTemplates(&templates.ListTemplatesParams{
-		Body: templates.ListTemplatesBody{
-			Reload: true,
-		},
-		Context: pmmapitests.Context,
+	t.Run("without pagination", func(t *testing.T) {
+		resp, err := client.ListTemplates(&templates.ListTemplatesParams{
+			Body: templates.ListTemplatesBody{
+				Reload: true,
+			},
+			Context: pmmapitests.Context,
+		})
+		require.NoError(t, err)
+
+		var found bool
+		for _, template := range resp.Payload.Templates {
+			if template.Name == name {
+				assert.Equal(t, expr, template.Expr)
```

Comment:
looks like we could check equality of two objects, can we?

Reply:
check `template` with some `expectedTemplate`

Reply:
I didn't touch this code, just moved to subtest, but sure, can it little refactor it